omnibus/package-scripts/updater-deb/postinst

```diff
@@ -77,6 +77,45 @@ install_method:
     echo "$install_info_content" > $CONFIG_DIR/install_info
 fi
 
+function generate_install_id() {
+  # Try generating a UUID based on /proc/sys/kernel/random/uuid
+  uuid=$(cat /proc/sys/kernel/random/uuid 2>/dev/null)
+  # If that does not work, then try uuidgen
+  if [ ${#uuid} -ne 36 ]; then
+    uuid=$(uuidgen 2>/dev/null)
+  fi
+  # Convert to lowercase
+  uuid=$(echo "$uuid" | tr '[:upper:]' '[:lower:]')
+  printf "$uuid"
+}
+
+function generate_install_signature() {
+  local install_id="$1"
+  local install_type="$2"
+  local install_time="$3"
+  printf "{\"install_id\":\"$install_id\",\"install_type\":\"$install_type\",\"install_time\":$install_time}"
+}
+
+install_id=$(generate_install_id)
+install_type="linux_manual_with_apt"
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏This likely should be different to track the usage of the newer updating product.